model-optimizer/extensions/front/tf/CTCGreedyDecoderReplacement.py

```diff
@@ -38,37 +70,35 @@ def pattern(**kwargs):
                    ('sparse_to_dense', dict(op='SparseToDense'))
                    ],
             edges=[('decoder', 'sparse_to_dense', {'out': 0}),
-                   ('decoder', 'sparse_to_dense', {'out': 2}),
                    ('decoder', 'cast', {'out': 1}),
-                   ('cast', 'sparse_to_dense', {'out': 0})
-                   ]
+                   ('cast', 'sparse_to_dense', {'out': 0})]
```

Comment:
I suggest to have just one transformation for CTCGreedyDecoder+SparseToDense because I see the common part in both patterns and one pattern is a sub-graph of the other.

Reply:
MO not allowed to detect pattern with different edge count. So we need to stay two pattern.